### PR TITLE
Add logger to piecestore server/node started log

### PR DESCRIPTION
**Pull request by cam-a**

### CODE REVIEW

File: pkg/piecestore/psserver/server.go [hunk 1]
```diff
@@ -103,10 +104,11 @@ type Server struct {
 	totalAllocated   int64
 	totalBwAllocated int64
 	verifier         auth.SignedMessageVerifier
+	log              *zap.SugaredLogger
```
Comment (aleitner) [inline pkg/piecestore/psserver/server.go]: Should be type `*zap.Logger`

File: pkg/piecestore/psserver/server.go [hunk 2]
```diff
@@ -103,10 +104,11 @@ type Server struct {
...
 }
 
 // Initialize -- initializes a server struct
-func Initialize(ctx context.Context, config Config, pkey crypto.PrivateKey) (*Server, error) {
+func Initialize(ctx context.Context, config Config, pkey crypto.PrivateKey, log *zap.SugaredLogger) (*Server, error) {
```
Comment (aleitner) [inline pkg/piecestore/psserver/server.go]: `*zap.Logger`

File: pkg/piecestore/psserver/server.go [hunk 3]
```diff
@@ -52,7 +52,7 @@ func (c Config) Run(ctx context.Context, server *provider.Provider) (err error)
 
 	ctx, cancel := context.WithCancel(ctx)
 
-	s, err := Initialize(ctx, c, server.Identity().Key)
+	s, err := Initialize(ctx, c, server.Identity().Key, zap.S())
```
Comment (aleitner) [inline pkg/piecestore/psserver/server.go]: change to a named logger
`zap.L().Named(fmt.Sprintf("storage node (%s)", server.Identity().ID))`

File: internal/testplanet/planet.go [hunk 4]
```diff
@@ -148,7 +148,7 @@ func New(t zaptest.TestingT, satelliteCount, storageNodeCount, uplinkCount int)
 			Path:               storageDir,
 			AllocatedDiskSpace: memory.GB.Int64(),
 			AllocatedBandwidth: 100 * memory.GB.Int64(),
-		}, node.Identity.Key)
+		}, node.Identity.Key, zap.S())
```
Comment (aleitner) [inline internal/testplanet/planet.go]: Change to a named logger
`zap.L().Named(fmt.Sprintf("storage node (%s)", node.Identity().ID))`

File: pkg/piecestore/psserver/server.go (hunk 1 above)
Comment (aleitner) [reply pkg/piecestore/psserver/server.go]: SugaredLogger can log formatted messages so we'll keep using that

File: pkg/piecestore/psserver/server.go (hunk 3 above)
Comment (aleitner) [reply pkg/piecestore/psserver/server.go]: SugaredLogger can log formatted messages so we'll keep using that

File: pkg/piecestore/psserver/server.go (hunk 2 above)
Comment (aleitner) [reply pkg/piecestore/psserver/server.go]: SugaredLogger can log formatted messages so we'll keep using that

File: internal/testplanet/planet.go (hunk 4 above)
Comment (cam-a) [reply internal/testplanet/planet.go]: the testplanet node has a named logger. I switched to that instead

**Comment by kaloyan-raev** [comment]
In my opinion the existing logs for storing, retrieving and deleting pieces are very helpful and they should not be removed. This is the major way to understand if something is really happening on the storage node. 

If the INFO level is too noisy then let's make them DEBUG.

File: pkg/piecestore/psserver/server.go [hunk 5]
```diff
@@ -105,10 +113,8 @@ type Server struct {
 	verifier         auth.SignedMessageVerifier
 }
 
-// Initialize -- initializes a server struct
-func Initialize(ctx context.Context, config Config, pkey crypto.PrivateKey) (*Server, error) {
-	dbPath := filepath.Join(config.Path, "piecestore.db")
-	dataDir := filepath.Join(config.Path, "piece-store-data")
+// NewEndpoint -- initializes a new endpoint for a piecestore server
+func NewEndpoint(ctx context.Context, config Config, db *psdb.DB, pkey crypto.PrivateKey, log *zap.Logger) (*Server, error) {
```
Comment (egonelbre) [inline pkg/piecestore/psserver/server.go]: ctx isn't needed here anymore, and move `log` as the first argument.

File: pkg/piecestore/psserver/server.go [hunk 6]
```diff
@@ -178,8 +180,9 @@ func Initialize(ctx context.Context, config Config, pkey crypto.PrivateKey) (*Se
 }
 
 // New creates a Server with custom db
-func New(dataDir string, db *psdb.DB, config Config, pkey crypto.PrivateKey) *Server {
+func New(dataDir string, db *psdb.DB, config Config, pkey crypto.PrivateKey, log *zap.Logger) *Server {
```
Comment (egonelbre) [inline pkg/piecestore/psserver/server.go]: Move `log` as the first argument.

**Pull request merged**